tests/utils.go

```diff
@@ -4898,3 +4898,20 @@ func IsLauncherCapabilityDropped(capability k8sv1.Capability) bool {
 	}
 	return false
 }
+
+// VMILauncher waiting for the VMI to be up but ignoring warnings like a disconnected guest-agent
+func VMILauncher(virtClient kubecli.KubevirtClient) func(vmi *v1.VirtualMachineInstance) *v1.VirtualMachineInstance {
+	return func(vmi *v1.VirtualMachineInstance) *v1.VirtualMachineInstance {
+		By("Starting a VirtualMachineInstance")
+		obj, err := virtClient.RestClient().Post().Resource("virtualmachineinstances").Namespace(NamespaceTestDefault).Body(vmi).Do(context.Background()).Get()
```

Comment:
any reason why you're using the RestClient here instead of the VirtualMachineInstance ?
```
vmi, err := virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Create(vmi)
```

**edit** leaving this because it's funny. You did this because you moved code I wrote, and i have no idea why I wrote it this way, lol.